tests/test_on_data/test_gin.py

```diff
@@ -185,6 +186,18 @@ class TestConverter(NWBConverter):
                 data_interface=BlackrockSortingExtractorInterface,
                 interface_kwargs=dict(file_path=str(DATA_PATH / "blackrock" / "FileSpec2.3001.nev")),
             ),
+            param(
+                data_interface=CellExplorerSortingInterface,
+                interface_kwargs=dict(
+                    folder_path=str(DATA_PATH / "cellexplorer" / "cell_explorer_example_sessionInfo.mat")
```

Comment:
```suggestion
                    file_path=str(DATA_PATH / "cellexplorer" / "cell_explorer_example_sessionInfo.mat")
```
CellExplorer uses `file_path` convention, which CI is complaining about now. Ideally it should only complain that it can't find the actual data at the specified location.